### PR TITLE
Allow filtering operations by date

### PR DESCRIPTION
**Pull request by kand**
Add ```from``` and ```to``` filters to operations endpoint so we can filter them by date.

### CODE REVIEW

**Comment by kand** [comment]
@MichaelButkovic @mparent61 @benghaziboy

File: bulbs/promotion/views.py [hunk 1]
```diff
@@ -65,8 +65,21 @@ def get(self, request, pzone_pk):
...
+                filters["when__gte"] = parsed
+
+        if "to" in request.GET:
+            parsed = dateparse.parse_datetime(request.GET["to"])
+            if parsed is not None:
+                filters["when__lt"] = parsed
+
         # get operations and serialize them
-        operations = PZoneOperation.objects.filter(pzone=pzone)
+        operations = PZoneOperation.objects.filter(**filters)
```
Comment (mparent61) [inline bulbs/promotion/views.py]: Nice

**Comment by mparent61** [comment]
:+1:

**Comment by MichaelButkovic** [comment]
LGTM

**Pull request merged**